app/Example.java

```diff
@@ -0,0 +1,50 @@
+import android.view.View;
+
+import io.ably.lib.realtime.AblyRealtime;
+import io.ably.lib.realtime.Channel;
+import io.ably.lib.types.AblyException;
+import io.ably.lib.types.Message;
+import io.ably.lib.types.Param;
+
+public class Example {
+
+	private static final String API_KEY = "INSERT-YOUR-API-KEY-HERE";
+
+	public void publishAndSubcscribe(Button buttonPublish, Button buttonHistory) {
+		AblyRealtime realtime = new AblyRealtime(API_KEY);
```

Comment:
This tutorial should really be done using the REST API, there is very little reason to use the Realtime API here.  Alternatively we could have two examples, one that runs like the Ruby one with REST